src/Evolution/DgSubcell/Matrices.cpp

```diff
@@ -67,16 +82,25 @@ const Matrix& projection_matrix(const Mesh<1>& dg_mesh,
                          Spectral::Quadrature::CellCentered>,
                      Spectral::maximum_number_of_points<
                          Spectral::Basis::FiniteDifference> +
-                         1>>(
-          [](const size_t local_num_dg_points,
-             const size_t local_num_fd_points) {
-            return Spectral::interpolation_matrix<Spectral::Basis::Legendre,
-                                                  Spectral::Quadrature::Gauss>(
-                local_num_dg_points,
-                Spectral::collocation_points<
-                    Spectral::Basis::FiniteDifference,
-                    Spectral::Quadrature::CellCentered>(local_num_fd_points));
-          });
+                         1>>([&subcell_quadrature](
+                                 const size_t local_num_dg_points,
+                                 const size_t local_num_fd_points) {
+        if (subcell_quadrature == Spectral::Quadrature::CellCentered) {
+          return Spectral::interpolation_matrix<Spectral::Basis::Legendre,
+                                                Spectral::Quadrature::Gauss>(
+              local_num_dg_points,
+              Spectral::collocation_points<Spectral::Basis::FiniteDifference,
+                                           Spectral::Quadrature::CellCentered>(
+                  local_num_fd_points));
+        } else {
```

Comment:
I do not like this if... else... statement, but
`Spectral::collocation_points<Spectral::Basis::FiniteDifference,
                                              subcell_quadrature>`
does not compile (with clang on wheeler)